helpers/utils.py

```diff
@@ -247,7 +247,7 @@ def get_kube_config() -> str:
 
 def get_kubectl_cmd() -> str:
     """Return the kubectl binary location within the snap, used to build k8s commands."""
-    kubectl_cmd = "{}/kubectl".format(os.environ["SNAP"])
+    kubectl_cmd = "{}/opt/spark/kubectl".format(os.environ["SNAP"])
```

Comment:
I'm just wondering whether kubectl should really be under opt/spark or rather some other places? I mean...it is really not connected to spark, is it? @taurus-forever ? Any suggestion?